core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/EntitlementInfos.android.kt

```diff
@@ -9,4 +9,4 @@ public actual typealias EntitlementInfos = RcEntitlementInfos
 public actual val EntitlementInfos.all: Map<String, EntitlementInfo>
     get() = all
 public actual val EntitlementInfos.verification: VerificationResult
-    get() = verification
\ No newline at end of file
+    get() = verification
```

Comment:
Hmm I'm wondering, what about the other properties for `EntitlementInfos` (like `active`, or `get`)? Maybe we still need to add them here but I wanted to check